nntrainer/layers/layer_normalization_layer.cpp

```diff
@@ -236,8 +235,8 @@ void LayerNormalizationLayer::calcDerivative(RunLayerContext &context) {
   TensorDim::TensorType weight_tensor_type =
     context.getWeight(wt_idx[LNParams::gamma]).getTensorType();
 
-  Tensor empty;
-  empty.setTensorType(weight_tensor_type);
+  Tensor empty =
+    Tensor("empty", weight_tensor_type.format, weight_tensor_type.data_type);
```

Comment:
We can set the format and data_type with weight_tensor_type. No need to set it. Also, we do not need to set the tensor name always.  I think when we create tensor like here, ```Tensor empty```, then it does not allocate the buffer. So.. I think it is ok to set the Tensor type here. The reason we set like this.. we do not want to allocate a buffer at this time.

Reply:
As we discussed offline, `Tensor empty;` would create FloatTensor as a default. 
I will make an additional Tensor constructor that only takes data type (e.g., `Tensor empty = Tensor(weight_tensor_type.data_type);`)